src/components/TransactionDetails/TransactionDetailsListItem.jsx

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable camelcase */
```

Comment:
could you explain why do you need it here please?

Reply:
My es-lint barks that my variable is not in camelCase.